euphonic/force_constants.py

```diff
@@ -1097,7 +1099,7 @@ def _dipole_correction_init(crystal: Crystal,
                 recip_q0 += recip_q0_tmp
             else:
                 break
-        vol = crystal._cell_volume()
+        vol = crystal.cell_volume().to('bohr^3').magnitude

```

Comment:
What is "magnitude" doing here? Is it a pint thing to return sans unit?

Reply:
Exactly, magnitude unwraps it to a bare array. This is why we usually chain it with `to` so we know what the unit will be and can use the values (relatively) safely.